public/js/chat.js

```diff
@@ -15,8 +15,11 @@ var Chat = function(id, player) {
 	//   self._parseUserData(JSON.parse(data));
 	// });
 
-	$.get("https://api.twitch.tv/kraken/videos/" + id + "?client_id=88bxd2ntyahw9s8ponrq2nwluxx17q", function(vodData) {
-		self.recordedTime = moment(vodData.recorded_at).utc();
+	//sending a client-id header by default
```

Comment:
Kind a nit-pick but can we put spaces after the `//` in the comments for consistency?